test/cpp/jit/test_gpu.cpp

```diff
@@ -4009,19 +4049,24 @@ TEST_F(NVFuserTest, FusionUnaryOps_CUDA) {
           std::make_tuple(std::make_pair(ValType::TensorView, dtype)));
     });
 
-    test_op(
-        /*blocks*/ 128,
-        /*threads*/ 64,
-        /*name*/ "rand_like",
-        /*Aten Func   */
-        [](std::array<IValue, 1>& vals) {
-          return at::rand_like(vals[0].toTensor());
-        },
-        /*JIT  Func   */
-        [](Val* in1) -> Val* { return unaryOp(UnaryOpType::RandLike, in1); },
-        /*Output      */ std::make_pair(ValType::TensorView, dtype),
-        /*Inputs Tuple*/
-        std::make_tuple(std::make_pair(ValType::TensorView, dtype)));
+    // TODO: why the rand_like test is failing for complex? Is it because each
+    // complex needs to draw 2 random numbers from the RNG? We need to enable
+    // this
```

Comment:
Yes, that could be the case. In these tests we're manually forcing the parallelization to try and match what eager mode does. I guess it's not quite the same for complex rand_like.

Reply:
Cross referencing: https://github.com/csarofeen/pytorch/pull/1481